compose/backend.yml

```diff
@@ -2,6 +2,35 @@
 version: '3'
 
 services:
+
+  annotation-api:
+    image: hmctspublic.azurecr.io/em/anno:latest
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://annotation-api-db:5432/emannotationapp
+      - SPRING_DATASOURCE_USERNAME=emannotationapp
+      - SPRING_DATASOURCE_PASSWORD=emannotationapp
+      - S2S_BASE_URI=http://service-auth-provider-api:8080
+      - IDAM_API_BASE_URI=http://idam-api:8080
+      - ENABLE_LIQUIBASE=true
+      - S2S_NAMES_WHITELIST=sscs,divorce,ccd,em_gw,jui_webapp,pui_webapp,em_npa_app,ccd_gw,ccd_gateway
+      - IDAM_API_BASE_URI=http://sidam-api:5000
+    links:
+      - annotation-api-db
```

Comment:
linking is an obsolete feature of Docker. It should no longer be used for new services. Please remove

Reply:
Removed the links.